examples/t14634.nim

```diff
@@ -0,0 +1,48 @@
+#[
```

Comment:
Under "examples"?

Reply:
Where then

Reply:
moved to tests/m14634.nim (not t14634.nim so it doesn't get run by testament)